src/plugins/github/fetchGithubRepo.js

```diff
@@ -188,6 +188,7 @@ function ensureNoMorePages(result: any, path = []) {
   }
   if (result.pageInfo) {
     if (result.pageInfo.hasNextPage) {
+      console.error(result);
```

Comment:
I’m okay with this, but only because the code is untested (though
there’s no reason that it should be).